src/webui/www/private/editfeedurl.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html lang="${LANG}">
+
+<head>
+    <meta charset="UTF-8">
+    <title>QBT_TR(Please type a RSS feed URL)QBT_TR[CONTEXT=RSSWidget]</title>
+    <link rel="stylesheet" href="css/style.css?v=${CACHEID}" type="text/css">
+    <script src="scripts/lib/MooTools-Core-1.6.0-compat-compressed.js"></script>
+    <script src="scripts/lib/MooTools-More-1.6.0-compat-compressed.js"></script>
+    <script src="scripts/misc.js?locale=${LANG}&v=${CACHEID}"></script>
+    <script>
+        "use strict";
+
+        new Keyboard({
+            defaultEventType: "keydown",
+            events: {
+                "Enter": (event) => {
+                    $("submitButton").click();
+                    event.preventDefault();
```

Comment:
`preventDefault` should come first:
```suggestion
                    event.preventDefault();
                    $("submitButton").click();
```
and the same to other event handlers below.

Reply:
FYI this is also how every other frame in `src/webui/www/private/` is written. I agree with your ordering though.

Reply:
>FYI this is also how every other frame in src/webui/www/private/ is written. 

We should probably change them.